atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -97,13 +97,36 @@ def run_in_orchestrator(self):
         compute their replacements and set build arg for worker.
 
         Exclude CSVs which already have a relatedImages section.
+
+        Returns operator metadata in format
+        related_images:
+          pullspecs:  # list of all related_images_pullspecs
+            - original: <original-pullspec1>  # original pullspec in CSV file
+              new: <new pullspec>   # new pullspec computed by this plugin
+              pinned: <bool>  # plugin pinned tag to digest
+              replaced: <bool>  # plugin modified pullspec (repo/registry/tag changed)
+            - original: ........
+          created_by_osbs: <bool>
         """
+        related_images_metadata = {
+            'pullspecs': [],
+            'created_by_osbs': True,
+        }
+        operator_manifests_metadata = {
+            'related_images': related_images_metadata
+        }
+
         operator_manifest = self._get_operator_manifest()
-        pullspecs = self._get_pullspecs(operator_manifest)
+        if operator_manifest.csv:
```

Comment:
Shall we fail build if there is no CSV file?

Reply:
I wanna say yes, but could we quickly verify that this won't break any existing builds?

Reply:
We can ask OLM devs if this will cause fatal failure. I'd say given it contains all the important metadata

Reply:
They agreed that CSV is the operator